jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -560,27 +561,24 @@
     trigger-job: ''
     timeout: 240
     jenkins_node: 'master'
-    runner: '{legacy-runner}'
-    provider-env: |
-        {aws-provider-env}
-        export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
-        export GINKGO_PARALLEL="y"
-        export PROJECT="k8s-jkns-e2e-aws"
-        export AWS_CONFIG_FILE='/var/lib/jenkins/.aws/credentials'
-        export AWS_SSH_KEY='/var/lib/jenkins/.ssh/kube_aws_rsa'
-        export KUBE_SSH_USER='admin'
-        # This is needed to be able to create PD from the e2e test
-        export AWS_SHARED_CREDENTIALS_FILE='/var/lib/jenkins/.aws/credentials'
+    runner: '{runner}'
```

Comment:
I think you can just drop this line, since it's the default. I'm seeing `{runner}` in the YAML.